docgen/type-aliases.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Is this the complete list of all external links?

Reply:
It is definitely a good question how we'll keep this file up to date as things change.  I'm giving myself a KR to write up the various TypeDoc procedures, and could add this there.

But for now, I think this is the full list of external types.